ql/src/test/results/clientpositive/llap/create_like2.q.out

```diff
@@ -59,3 +59,285 @@ Bucket Columns:     	[]
+empno               	int                 	                    
+name                	string              	                    
+	 	 
+# Partition Information	 	 
+# col_name            	data_type           	comment             
+dept                	string              	                    
+	 	 
+# Detailed Table Information	 	 
+Database:           	default             	 
+#### A masked pattern was here ####
+Retention:          	0                   	 
+#### A masked pattern was here ####
+Table Type:         	EXTERNAL_TABLE      	 
+Table Parameters:	 	 
+	COLUMN_STATS_ACCURATE	{\"BASIC_STATS\":\"true\"}
+	EXTERNAL            	TRUE                
+	bucketing_version   	2                   
+	discover.partitions 	true                
+	numFiles            	0                   
+	numPartitions       	0                   
+	numRows             	0                   
+	rawDataSize         	0                   
+	totalSize           	0                   
+#### A masked pattern was here ####
+	 	 
+# Storage Information	 	 
+SerDe Library:      	org.apache.hadoop.hive.ql.io.orc.OrcSerde	 
+InputFormat:        	org.apache.hadoop.hive.ql.io.orc.OrcInputFormat	 
+OutputFormat:       	org.apache.hadoop.hive.ql.io.orc.OrcOutputFormat	 
+Compressed:         	No                  	 
+Num Buckets:        	-1                  	 
+Bucket Columns:     	[]                  	 
+Sort Columns:       	[]                  	 
+PREHOOK: query: create table test_mm1 like test_external
+PREHOOK: type: CREATETABLE
+PREHOOK: Output: database:default
+PREHOOK: Output: default@test_mm1
+POSTHOOK: query: create table test_mm1 like test_external
+POSTHOOK: type: CREATETABLE
+POSTHOOK: Output: database:default
+POSTHOOK: Output: default@test_mm1
+PREHOOK: query: desc formatted test_mm1
+PREHOOK: type: DESCTABLE
+PREHOOK: Input: default@test_mm1
+POSTHOOK: query: desc formatted test_mm1
+POSTHOOK: type: DESCTABLE
+POSTHOOK: Input: default@test_mm1
+# col_name            	data_type           	comment             
+empno               	int                 	                    
+name                	string              	                    
+	 	 
+# Partition Information	 	 
+# col_name            	data_type           	comment             
+dept                	string              	                    
+	 	 
+# Detailed Table Information	 	 
+Database:           	default             	 
+#### A masked pattern was here ####
+Retention:          	0                   	 
+#### A masked pattern was here ####
+Table Type:         	MANAGED_TABLE       	 
+Table Parameters:	 	 
+	COLUMN_STATS_ACCURATE	{\"BASIC_STATS\":\"true\"}
+	bucketing_version   	2                   
+	numFiles            	0                   
+	numPartitions       	0                   
+	numRows             	0                   
+	rawDataSize         	0                   
+	totalSize           	0                   
+	transactional       	true                
+	transactional_properties	insert_only         
```

Comment:
This is a bit counterintuitive to me. Why is this table property set to insert only whereas in the other case it is default?

Reply:
In SemanticAnalyzer, if there is no UserSpecifiedFormat in CTLT, while adding all default properties we are using TextFile Input/Output classes because default hive.default.fileformat.managed is TextFile. But org.apache.hadoop.hive.ql.ddl.table.create.like.CreateTableLikeOperation#createTableLikeTable is validating whether any UserSpecifiedFormat, if not its using same table object which is set as ORC. I will change my testcase to set hive.default.fileformat.managed=ORC which should convert it to default & will try to fix that issue in another jira.